### DISCUSSION TITLE
Is `Luvref` document wrong?

### DISCUSSION THREAD

**Discussion by linrongbin16** [neovim/neovim#25256] [Q&A] [answered]
In https://neovim.io/doc/user/luvref.html, the sample code is writing with:

```lua
vim.uv
```

But in my nvim-v0.9.2, there's no such module, there's only `vim.loop`.

Is the doc wrong?

**Comment by zeertzjq** [comment, marked as answer]
It's newer than your Nvim version. Use `:h luvref.txt` instead.

**Reply by linrongbin16** [reply]
BTW, is `vim.uv` just a renaming, or the internal feature/component is different? can I still use `vim.loop` in newer version of nvim?

**Reply by zeertzjq** [reply]
> BTW, is `vim.uv` just a renaming?

Yes.

> Can I still use `vim.loop` in newer version of nvim?

Yes.

**Reply by linrongbin16** [reply]
thank you!
